src/main/java/net/imagej/legacy/plugin/MacroLanguageSupportPlugin.java

```diff
@@ -143,6 +143,7 @@ public MacroAutoCompletionKeyListener(final AutoCompletion ac,
 			final RSyntaxTextArea textArea)
 		{
 			this.ac = ac;
+			this.ac.setAutoCompleteSingleChoices(false);
```

Comment:
Ah cool! That fixes an old bug. Thanks for finding out how to do this! ❤️

Reply:
Thanks! It took me a while to trace it. Anyways, it is nice to have single choices autocompleted, but it opened a second issue that I haven't had the time to track down and zap: it annoyingly suggests the text that's beeing edited when typing a function for the first time, or when editing an already existing variable declaration:

![annoying autocomplete](https://user-images.githubusercontent.com/9939186/69306746-5a72d180-0c07-11ea-9945-d3c42f4922a4.png)

I have some ideas to start, but will have to spend some extra time digging in the autocompletitionProvider class to see if it can be implemented as I think.

Reply:
I solved this issue with e99640d 😅